ethpm/tools/builder.py

```diff
@@ -379,8 +399,27 @@ def normalize_contract_type(
             yield "runtime_bytecode", normalize_bytecode_object(
                 contract_type_data["evm"]["deployedBytecode"]
             )
-    # todo support natspec.
-    yield "natspec", {}
+    if any(key in contract_type_data for key in ["devdoc", "userdoc"]):
```

Comment:
*nitpick*

1. When doing checks for inclusion you should use a set `{"devdoc", "userdoc"}` as they are better suited for inclusion checks.
2. The value should probably be elevated to a top level constant `NATSPEC_FIELDS = {'devdoc', 'userdoc'}`